framework/src/st_things/st_things_request_handler.c

```diff
@@ -355,18 +356,18 @@ static bool get_supported_properties(const char *res_type, const char *res_uri,
 		// 2. Get the properties for each resource type.
 		props = (things_attribute_info_s ***) things_calloc(type_count, sizeof(things_attribute_info_s **));
 		if (NULL == props) {
-			THINGS_LOG_E(THINGS_ERROR, TAG, "Failed to allocate memory for resource properties.");
+			THINGS_LOG_E(TAG, "Failed to allocate memory for resource properties.");
 			goto EXIT_WITH_ERROR;
 		}
 		prop_count = (int *)things_calloc(type_count, sizeof(int));
 		if (NULL == prop_count) {
-			THINGS_LOG_E(THINGS_ERROR, TAG, "Failed to allocate memory for resource properties.");
+			THINGS_LOG_E(TAG, "Failed to allocate memory for resource properties.");
 			goto EXIT_WITH_ERROR;
 		}
 
 		for (int index = 0; index < type_count; index++) {
 			if (!things_get_attributes_by_resource_type(res_types[index], &prop_count[index], &props[index])) {
-				THINGS_LOG_V(THINGS_ERROR, TAG, "Failed to get the properties of resource type (%s).", res_types[index]);
+				THINGS_LOG_V(TAG, "Failed to get the properties of resource type (%s).", res_types[index]);
```

Comment:
this is error log. 
so , in my opinion, It seems that changing from THINGS_LOG_V to THINGS_LOG_E is better.

Reply:
ok.